features/optee/optee.cfg

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: MIT
```

Comment:
We do have features/tee/tee.cfg, maybe we can just replace with this one?

Reply:
Yes, I'm planning to switch to the mainline version with our changes later.
Are we going to keep this PR as simple as possible?

Reply:
I dropped it in this PR.